timezone_converter/comparison_view.py

```diff
@@ -55,7 +55,14 @@ def _build_table(self) -> Table:
         fmt = '%Y-%m-%d %H:%M'
 
         current_hour = datetime.now().hour
-        for hour in range(24):
+        if single_hour is None:
+            hour_range = range(24)
+        elif single_hour == 'now':
+            hour_range = range(current_hour, current_hour + 1)
+        else:
+            hour_range = range(int(single_hour), int(single_hour) + 1)
+
+        for hour in hour_range:
```

Comment:
As `current_hour` would only be used on this for loop, I would prefer that we move the `current_hour` declaration to the line before this one (line 64)